### DISCUSSION TITLE
welcome-emo-desktop-pet/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#748] [General]
# welcome-emo-desktop-pet/

以前、以下の記事を見かけて一目ぼれし、当時募集していたIndiegogoというクラウドファンディングサイトで注文したのですが、強制キャンセルとなってしまったため公式直販で3月頃に注文しました。

https://joy-lab.net/welcome-emo-desktop-pet/

**Comment by joy-power** [comment]
テスト
